src/validation.h

```diff
@@ -709,6 +709,9 @@ class Chainstate
     bool ConnectBlock(const CBlock& block, BlockValidationState& state, CBlockIndex* pindex,
                       CCoinsViewCache& view, bool fJustCheck = false) EXCLUSIVE_LOCKS_REQUIRED(cs_main);
 
+    bool SpendBlock(const CBlock& block, const CBlockIndex* pindex, const uint256& block_hash,
```

Comment:
2be3c59151bcb840eb8cbd045a3dada6775ebc9e: this or the final commit would be a good time to document both `SpendBlock` and `ConnectBlock`.

Reply:
Yes, that is the gist of it. Will add a commit at the end to add some documentation in the header.

Reply:
Added a commit.